onboarding.adoc

```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
 Even for seasoned open source contributors, when they join a new community they will need to get familiar with new tools, new processes, and even new terminologies.
 Depending on the community, seemingly simple terms like contributions, community members, projects, upstream, etc.
 may mean something slightly different and can easily cause confusion for new community members.
-  
+
+== Onboarding basics
+
+As you work to make your project's onboarding process and experience most effective, ask yourself these questions:
+
+- What can contributors add to the project?
```

Comment:
I feel like there's too much emphasis on `add` here. Someone could contribute to the project by archiving or removing outdated content (and help with house cleaning). Maybe rephrase to something like "What can community members contribute to the project" (and something similar to the rest of the bullets)?

Reply:
Great point. Let me see what I can do.

Reply:
What do you think of this?

Reply:
👍